.github/workflows/release.yml

```
@@ -152,3 +152,8 @@ jobs:
             *.vsix
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publish to VS Code Marketplace
+        run: npx @vscode/vsce publish --no-dependencies --packagePath *.vsix
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:1:58: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)